### PR TITLE
Adding nvidia-gpu-tools.py

### PR DESCRIPTION
**Pull request by visheshtanksale**
Updated the utility that sets CC mode on GPU. 

Fixes: https://nvbugspro.nvidia.com/bug/4348282

### CODE REVIEW

**Comment by visheshtanksale** [comment]
@zvonkok @shivamerla @tariq1890 can you please take a look. Some how I am not able to add any reviewers to this PR.

File: deployments/container/Dockerfile.ubi8 [hunk 1]
```diff
@@ -51,10 +52,10 @@ RUN mkdir /licenses && mv /NGC-DL-CONTAINER-LICENSE /licenses/NGC-DL-CONTAINER-L
 COPY --from=build /artifacts/k8s-cc-manager /usr/bin/k8s-cc-manager
 COPY --from=build /artifacts/kubectl /usr/bin/kubectl
 
+ARG GPU_ADMIN_TOOLS_VERSION
 # Setup CC tools and licenses
 COPY --from=build /artifacts/cc-manager.sh /usr/bin/cc-manager.sh
-RUN curl -o /usr/bin/gpu_cc_tool.py -L https://raw.githubusercontent.com/NVIDIA/nvtrust/main/host_tools/python/gpu_cc_tool.py && chmod a+x /usr/bin/gpu_cc_tool.py
-RUN curl -o /licenses/LICENSE.txt -L https://raw.githubusercontent.com/NVIDIA/nvtrust/main/host_tools/python/LICENSE.txt
+RUN git clone https://github.com/NVIDIA/gpu-admin-tools.git /usr/bin/gpu-admin-tools && chmod a+x /usr/bin/gpu-admin-tools/nvidia_gpu_tools.py
```
Comment (shivamerla) [inline deployments/container/Dockerfile.ubi8:58]: where is GPU_ADMIN_TOOLS_VERSION used here? Were you planning to checkout the specific tag?

Comment (elezar) [reply deployments/container/Dockerfile.ubi8:58]: Does it make sense to curl just the py file from a versioned URL? Are there other files required in the repo? (also, should the `.py` file not already be executable when cloned?)

Comment (visheshtanksale) [reply deployments/container/Dockerfile.ubi8]: > where is GPU_ADMIN_TOOLS_VERSION used here? Were you planning to checkout the specific tag?

@shivamerla Yes I forgot to use it in the git clone. Fixed it.

Comment (visheshtanksale) [reply deployments/container/Dockerfile.ubi8]: > Does it make sense to curl just the py file from a versioned URL? Are there other files required in the repo? (also, should the `.py` file not already be executable when cloned?)

@elezar Cloning the repo becase there are other dependencies in that repo. Removed the `chmod`

File: deployments/container/Dockerfile.ubi8 [hunk 2]
```diff
@@ -37,6 +37,7 @@ FROM nvidia/cuda:${CUDA_VERSION}-base-${BASE_DIST}
 
 # Install Python Interpreter
 RUN dnf install -y python38 && pip3 install numpy
+RUN dnf install -y git
```
[the remark this answers is not in the document]
Comment (visheshtanksale) [reply deployments/container/Dockerfile.ubi8]: Thanks for catching it. Fixed!

**Comment by shivamerla** [comment]
Thanks @visheshtanksale were you able to verify with latest CC manager container?

File: deployments/container/Dockerfile.ubi8 (hunk 2 above)
[the remark this answers is not in the document]
Comment (tariq1890) [reply deployments/container/Dockerfile.ubi8]: Thanks

**Comment by visheshtanksale** [comment]
> Thanks @visheshtanksale were you able to verify with latest CC manager container?

@shivamerla Yes I tested it by setting the CC mode on cluster-policy object and verified the mode is set on GPU by running nvidia-gpu-tools.py on actual host

**Comment by zvonkok** [comment]
@visheshtanksale Did you test all three modes ?  cc=off, cc=on , cc=devtools and going back to cc=on or cc=off or cc=devtools?

**Comment by visheshtanksale** [comment]
> @visheshtanksale Did you test all three modes ? cc=off, cc=on , cc=devtools and going back to cc=on or cc=off or cc=devtools?

@zvonkok yes tested switching around between all three modes.

**Review by shivamerla — APPROVED** [review]

**Review by tariq1890 — APPROVED** [review]

**Pull request merged**